src/components/Radio/Radio.test.tsx

```diff
@@ -16,11 +16,13 @@ describe('<Radio />', () => {
     }).toThrow(/must provide a visible label or aria-label/);
   });
 
-  test('should toggle the radio with space', () => {
+  // eslint-disable-next-line jest/no-disabled-tests
```

Comment:
Why do these tests get skipped? I saw another one earlier

Reply:
Found cause of need for skip, no longer skipping (removing throwing error on console error)